src/fork_gc.c

```diff
@@ -772,6 +772,8 @@ static FGCError FGC_parentHandleTerms(ForkGC *gc) {
   if (idxKey) {
     RedisModule_CloseKey(idxKey);
   }
+  // TODO: check if this flow always yields a succeful write.
+  IndexSpec_UpdateVersion(sp);
```

Comment:
In the case of GC, the update of the version is not necessary
GC does not modify the "version" of the data, just removes garbage

Reply:
greate